handlers/userform-create.js

```diff
@@ -135,6 +138,8 @@ module.exports = {
             req.log("created form:", form.uuid);
             cb(null, form);
 
+            if (newForm.scopeQuery) return;
```

Comment:
So do we not broadcast new inbox items if they have a `scopeQuery` defined?  

Will they only see the inbox item when they refresh the browser / mobile app?

Bring this up with the team to see if that is the design we want.

However, if we want to send out broadcast updates on the correct users, then we need to:
- gather all `potentialUsers` = ( `users` + usersFromRoles(`roles`) )
- for each `potentialUsers`: if they pass the scopeQuery, then add them to the `listOfUpdateUsers`
- then perform `await req.broadcast.inboxCreate(listOfUpdateUsers, null, form);`

to do this, you should pull out the `isCorrectProcesForm()` logic out of `inbox-find` and put it in a shared `utility` that both `inbox-find` and `userform-create` can use.

Reply:
> So do we not broadcast new inbox items if they have a `scopeQuery` defined?

I'm ok with this behavior short term. I'd like to get this useable first and then come back to this.

Actually we need to figure this out for our model updates as well. Currently we don't broadcast if the scope has inUser query. It might be good to tackle these together, soon.